project-fortis-interfaces/src/components/Admin/Streams/StreamEditor.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { DataGrid } from '../DataGrid';
+import { getColumns } from '../shared';
+import CreateStream from './CreateStream';
+import StreamParamsButtonFormatter from './StreamParamsButtonFormatter';
+import StreamStatusButtonFormatter from './StreamStatusButtonFormatter';
+import { Card, CardTitle, CardText } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+
+class StreamEditor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.remove = this.remove.bind(this);
+
+    this.state = {
+      streamToBeEdited: {}
+    }
+  }
+
+  componentDidMount() {
+    this.props.flux.actions.ADMIN.load_streams();
+  }
+
+  getStreamColumns() {
+    const columnValues = [
+      {key: "pipelineKey", name: "Type"},
+      {width: 115, key: "status", name: "Status", formatter: <StreamStatusButtonFormatter flux={this.props.flux}/>, getRowMetaData: (row) => row},
+      {width: 70, key: "params", name: "Edit", formatter: <StreamParamsButtonFormatter flux={this.props.flux}/>, getRowMetaData: (row) => row}
+    ];
+
+    return getColumns(columnValues);
+  }
+
+  remove = streams => {
+    this.prepareStreamsForSave(streams);
+    this.props.flux.actions.ADMIN.remove_streams(streams);
+  }
+
+  prepareStreamsForSave = streams => {
+    streams.forEach(stream => {
+      if (typeof stream.params === 'string') stream.params = JSON.parse(stream.params);
```

Comment:
Can use lodash's [isString](https://lodash.com/docs/4.17.5#isString) here instead of typeof.